UnitedIncome/components
# InfoBox & InfoBoxFormattedRows -> TypeScript

### Description
> Please provide a brief description of your changes below.

TypeScript-ified the `InfoBox` and `InfoBoxFormattedRows` components.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Pull down the branch and ensure the components still work and behave as expected locally.

### Comments
> Any additional comments that will aid with the review process.

A few strange things I encountered:

1. In testing, I have a test that returns an array of `ReactNode`s that I try to find the `.length` of. TS did not like this unless I casted the variable initially to type `any`. This will undoubtedly come up in other parts of the component library and `client` as well so we should figure out what the best way to address this is.
2. `InfoBoxFormattedRows` has a property (`rowData`) that is an array of objects. I was able to type it correctly but I don't think the expected properties of each object shows up appropriately in the Storybook docs. Hopefully there is a workaround for this but I wasn't able to find one in my brief ~15 mins of searching.
